Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugSystemComponent.cpp

```diff
@@ -499,8 +542,145 @@ namespace Multiplayer
+                            m_filteredAuditTrail.push_back(*elem);
+                        }
+
+                        AZStd::pair<AZStd::string, AZStd::string> cliServValues =
+                            elem->m_children.front().m_elements.front()->GetClientServerValues();
+                       if(AZStd::string::format(
+                            "%d %d %s %s", static_cast<uint16_t>(elem->m_inputId), static_cast<uint32_t>(elem->m_hostFrameId), cliServValues.first.c_str(), cliServValues.second.c_str()).contains(filter))
+                       {
+                           m_filteredAuditTrail.push_back(*elem);
+                       }
+
+                    }
+                }
+                // Desyncs and inputs can contain multiple line items
+                else
+                {
+                    if (AZStd::string::format(nodeTitle, elem->m_name.c_str()).contains(filter))
+                    {
+                        m_filteredAuditTrail.push_back(*elem);
+                        continue;
+                    }
+
+                    if (AZStd::string::format("%d %d", static_cast<uint16_t>(elem->m_inputId), static_cast<uint32_t>(elem->m_hostFrameId))
+                            .contains(filter))
+                    {
+                        m_filteredAuditTrail.push_back(*elem);
+                        continue;
+                    }
+
+                    // Attempt to construct a filtered input
+                    Multiplayer::AuditTrailInput filteredInput(elem->m_category, elem->m_inputId, elem->m_hostFrameId, elem->m_name, {});
+
+                    for (const auto& child : elem->m_children)
+                    {
+                        if (child.m_name.contains(filter))
+                        {
+                            filteredInput.m_children.push_back(child);
+                        }
+                        else if (child.m_elements.size() > 0)
+                        {
+                            MultiplayerAuditingElement filteredChild;
+                            filteredChild.m_name = child.m_name;
+
+                            for (const auto& childElem : child.m_elements)
+                            {
+                                AZStd::pair<AZStd::string, AZStd::string> cliServValues = childElem->GetClientServerValues();
+                                if (AZStd::string::format(
+                                        "%s %s %s", childElem->GetName().c_str(), cliServValues.first.c_str(), cliServValues.second.c_str())
+                                        .contains(filter))
+                                {
+                                    filteredChild.m_elements.push_back(childElem.get()->Clone());
+                                }
+                            }
+
+                            if (filteredChild.m_elements.size() > 0)
+                            {
+
+                            filteredInput.m_children.push_back(filteredChild);
```

Comment:
There's a bunch of weird formatting going on throughout these changes.. I'm not entirely sure what happened, but it's worth a quick pass to clean up anything that got a bit munged..

Reply:
Hmm yeah, this function has some oddities. I've cleaned them up.